### PR TITLE
Wtc 92 remove lib

### LINKED ISSUE #92: Remove lib directory

### PR DESCRIPTION
**Pull request by skibinska**
fixes #92 
related #58 

- Remove lib directory
- Add materialize-css and jquery to package.json
- Update default.hbs
- Update routes.js
- Add tests

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -88,10 +88,10 @@ module.exports = [
   },
   {
     method: 'GET',
-    path: '/lib/{file*}',
+    path: '/node_modules/{file*}',
     handler: {
       directory: {
-        path: 'lib'
+        path: 'node_modules'
```
Comment (des-des) [inline src/routes.js]: Can you build a route for each of your deps?
ie `public/materialize.css`.

Comment (des-des) [inline src/routes.js]: or public/materialise/**

Comment (skibinska) [inline src/routes.js]: Hi @des-des, materialize-css is in node_modules folder not in a public, should I move it to the public folder?

Comment (des-des) [inline src/routes.js]: @skibinska your endpoints do not need to be coupled to your file system.
ie
```js
method: 'GET',
path: '/public/materialise.css',
  handler: {
    file: {
      path: 'node_modules/materialise/dist/materialise.min.css'
    }
  }
}
```

Comment (des-des) [inline src/routes.js]: This has two advantages.
  1. Less coupling. If your files move around on the server, you do not need to change the urls in the client. Ie the client should not need knowledge of the file structure on the server to get resources, it is the job of your server to make these available in a coherent way.
  2. You do not need to expose your whole node_modules folder (poss 10s of thousands of files)

Comment (skibinska) [inline src/routes.js]: make sense! thanks @des-des :+1:

**Pull request merged**